app/models/course/group.rb

```diff
@@ -39,4 +40,11 @@ def remove_duplicate_users
       group_users.delete(group_user)
     end
   end
+
+  def users_are_unique
+    new_group_users = group_users.select(&:new_record?)
+    if new_group_users.count != new_group_users.uniq { |group_user| group_user.user }.count
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.<br>Pass `&:user` as an argument to `uniq` instead of a block.